SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Channel/Channel.cs

```diff
@@ -635,11 +648,11 @@ public void SetNetworkRequestAllowed(bool isAllowed)
         {
             if (isAllowed)
             {
-                Resume(_mutex.State, false);
+                Resume(_mutex.State, false).ConfigureAwait(false);
```

Comment:
@dahliaingabire-okta , I think we don't need `.ConfigureAwait(false)` if we don't `await` it. Same for the two places below.

Reply:
Right! Removed; thank you! Will you review again?

Reply:
Sorry for the delay. Approved. There are some checks failed, I'll take a look and get back to you.